aggregator.go

```diff
@@ -305,6 +307,63 @@ func printEnv() int {
 	return ExitStatusOK
 }
 
+func performMigrations() int {
+	migrationArgs := os.Args[2:]
+
+	db, err := createStorage()
+	if err != nil {
+		log.Error().Err(err).Msg("Unable to prepare DB for migrations")
+		return ExitStatusPrepareDbError
+	}
+	defer closeStorage(db)
+
+	dbConn := db.GetConnection()
+
+	if err := migration.InitInfoTable(dbConn); err != nil {
+		log.Error().Err(err).Msg("Unable to initialize migration info table")
```

Comment:
forgot return?

Reply:
@Sergey1011010 Yes, thanks, I knew I definitely missed one of those on-error returns.